enclave-runner/Cargo.toml

```diff
@@ -29,6 +30,11 @@ lazy_static = "1.2.0"                           # MIT/Apache-2.0
 libc = "0.2.48"                                 # MIT/Apache-2.0
 nix = "0.13.0"                                  # MIT
 openssl = { version = "0.10", optional = true } # Apache-2.0
+crossbeam = "0.7.1"                             # MIT/Apache-2.0
+num_cpus = "1.10.0"                             # MIT/Apache-2.0
+tokio = "0.1.22"                                # MIT
+futures-preview = { version = "0.3.0-alpha.17", features = ["compat", "io-compat"] }
```

Comment:
Is it possible to use the latest version of futures now without much change to the code? If so, please update this to use https://crates.io/crates/futures version 0.3.4